code/modules/client/client procs.dm

```diff
@@ -219,6 +219,9 @@
 	if(!tooltips)
 		tooltips = new /datum/tooltip(src)
 
+	if(map.base_turf == /turf/snow)
```

Comment:
I don't see this is hardcoding, if it's a snowmap it's a snowmap, do the snowmap things